docs/en/api/POSIX-API-local-cache.md

```diff
@@ -0,0 +1,438 @@
+- [entry_timeout](https://manpages.debian.org/testing/fuse/mount.fuse.8.en.html#entry_timeout=T): Specifies the timeout in seconds for which directory listing results are cached. The default is 1.0 second.
+
+The timeout time can be enlarged via Fuse mount command:
+```console
+$ bin/alluxio-fuse mount ufs_path mount_point -o attr_timeout=600 -o entry_timeout=600 
+```
+
+Recommend to set the timeout values based on the following factors:
+- Memory resources. The longer the timeout, the more metadata may be cached by kernel which contributes to higher memory consumption.
+  One can pre-decide how much memory to allocate to metadata kernel cache first.
+  Monitor the actual memory consumption while setting a large enough timeout value.
+  Then decide the timeout value suitable for the target memory usage.
+- Dataset in-use time. If the timeout value is bigger than the whole dataset in-use time and there are enough available memory resources,
+  cache invalidation and refresh will not be triggered, thus the highest cache-hit ratio and best performance can be achieved.
+- Dataset size. Kernel metadata cache for a single file takes around 300 bytes (up to 1KB), 3GB (up to 10GB) for 10 million files.
+  If the memory space needed for caching the metadata of the whole dataset is much smaller than the available memory resources,
+  recommend setting the timeout to your dataset in-use time. Otherwise, you may need to trade-off between memory consumption and cache-hit ratio.
+
+Note that, even a short period (e.g. `timeout=60` or `timeout=600`) of kernel metadata cache may significantly improve the overall metadata read performance and/or data read performance.
+Test against your common workloads to find out the optimal value.
+
+{% endnavtab %}
+{% navtab Userspace Metadata Cache Configuration %}
+
+Userspace metadata cache can be enabled via
+```console
+$ bin/alluxio-fuse mount ufs_path mount_point -o metadata_cache_size=<size> -o metqadata_cache_expire=<timeout>
+```
+`metadata_cache_size` (Default = `0` which means disabled): Maximum number of entries in the metadata cache. Each 1000 entries cause about 2MB memory.
+`metadata_cache_expire` (Default = `10min`): Specify expire time for entries in the metadata cache
+
+The metadata is cached in the AlluxioFuse Java process heap so make sure `metadata_cache_size * 2KB * 2 < AlluxioFuse process maximum memory allocated`.
+For example, if AlluxioFuse is launched with `-Xmx=16GB` and metadata cache can use up to 8GB memory, then `metadata_cache_size` should be smaller than 4 million.
+
+{% endnavtab %}
+{% endnavtabs %}
+
+#### Data Cache
+
+{% navtabs dataCache %}
+{% navtab Kernel Data Cache Configuration %}
+
+FUSE has the following I/O modes controlling whether data will be cached and the cache invalidation policy:
+- `direct_io`: disables the kernel data cache. Does not support in AlluxioFuse with libfuse3 yet.
```

Comment:
I am a little confused.

Do you mean, the "kernel data cache" is not supported in Alluxio with libfuse3?
What about "kernel data cache" in Alluxio with libfuse2?

So,  this 'direct_io' should always be used? Or we should make it as default?
And when it is ready, we can make this optional based on user mount option?

Reply:
`direct_io` which by passing kernel data cache has different configuration way in libfuse 2 and libfuse 3.

Libfuse 2, it can be set as a global mount option  `alluxio-fuse mount .... -o direct_io`
but with libfuse3, it cannot be set as a global mount option but a configuration for each open() and create() operation 
```
  public int open(String path, FuseFileInfo fi) { <-- the FuseFileInfo contains information e.g. direct_io or not
```
So libfuse 3 does support `direct_io` but our libfuse 3 alluxio fuse implementation haven't support yet, will see how to make it more clear.

kernel_cache auto_cache mount option is the same between libfuse 2 and libfuse 3